<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDlg</class>
 <widget class="QDialog" name="DownloadDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownloadDlg</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>华文行楷</family>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>正在下载更新程序</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>文件大小/已下载:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sizeLabel">
       <property name="text">
        <string>00.00MB/00.00MB</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>http://download.osgeo.org/libtiff/old/tiff-3.7.0beta.zip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="speedLabel">
       <property name="text">
        <string>0.00Mb/s</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="abortBtn">
       <property name="text">
        <string>终止</string>
       </property>
       <property name="icon">
        <iconset resource="../Resource.qrc">
         <normaloff>:/images/Resources/images/del.ico</normaloff>:/images/Resources/images/del.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
